api-spec/protobuf/ark/v1/service.proto

```diff
@@ -123,11 +109,13 @@ message GetBoardingAddressResponse {
 
 /* In-Round Transaction API messages */
 
-message RegisterInputsForNextRoundRequest {
-  repeated Input inputs = 1;
+message RegisterIntentRequest {
```

Comment:
lets not replace it, and only add a new RPC

Reply:
> lets not replace it, and only add a new RPC

Ideally we do remove it in the medium term, it's a redundant rpc  and keeps duplicate logic around input proofs